src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {
+      movies.map(el => (
+        <MovieCard movie={el} />
```

Comment:
```suggestion
      movies.map(movie => (
        <MovieCard movie={movie}  key={movie.imdbId}/>
```
consider more meaningful variable
add key prop